### PR TITLE
Query and update animals table

### PR DESCRIPTION
**Pull request by Achilles-Dev**
- [ ] Added some data to the animals' table.
- [ ] Updated the animals' table schema.
- [ ] Added some queries using transaction.
- [ ] Added queries to answer some questions.

### CODE REVIEW

File: schema.sql [hunk 1]
```diff
@@ -7,5 +7,6 @@ CREATE TABLE animals (
     escape_attempts INT,
     neutered BOOLEAN,
     weight_kg DECIMAL,
+    species VARCHAR(100),
     PRIMARY KEY (id)
 );
```
Comment (KDlamini) [inline schema.sql:11]: Please add a new column called `species` with a string type in the schema.sql file as per project requirements.
image

```
ALTER TABLE animals ADD species varchar(255);
```

File: queries.sql [hunk 2]
```diff
@@ -14,4 +14,52 @@ SELECT * FROM animals WHERE neutered = True;
...
+
+UPDATE animals SET weight_kg = weight_kg * -1 WHERE weight_kg < 0;
+
+COMMIT;
+
+-- Answering questions with queries
+
+SELECT COUNT(*) FROM animals;
+
+SELECT COUNT(*) FROM animals WHERE escape_attempts = 0;
```
Comment (KDlamini) [inline queries.sql:65]: You are missing four more queries highlighted in the project requirements below:
image

- I will give you an example of the first two:

What is the average weight of animals?

```
   SELECT AVG(weight_kg) FROM animals;
```
Who escapes the most, neutered or not neutered animals?

```
   SELECT neutered, AVG(escape_attempts) FROM animals
   GROUP BY neutered;
```

- Kindly finish the rest of the queries by writing the SQL for `What is the minimum and maximum weight of each type of animal?` and `What is the average number of escape attempts per animal type of those born between 1990 and 2000?`.

**Review by KDlamini — CHANGES_REQUESTED** [review]
### Hi @Achilles-Dev 🙋‍♀️,  

Great work so far! 💯 
There are some issues that you still need to work on to improve your project according to the requirements. You are almost there! 💪🏾 

pugReaching

## Required Changes ♻️
 
**Kindly look through the comments below 👇 for required changes**

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
N/A

Cheers 🥂  and Happy coding!!! 👯 
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @KDlamini  in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: queries.sql (hunk 2 above)
Comment (Achilles-Dev) [reply queries.sql:65]: Thank you for the explanation. I did everything but somehow did not save it. I'm grateful for your examples.

**Review by gemmen29 — APPROVED** [review]
Hi @Achilles-Dev, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## To Highlight 💯 
- Descriptive pull request ✔️ 
- Descriptive commit messages ✔️ 



## Optional suggestions

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**